Functions/PollEmailProcessing.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Functions
+{
+    public class PollEmailProcessing
+    {
+        private readonly ILogger _logger;
```

Comment:
It should be` private readonly ILogger<PollEmailProcessing> _logger;`
i.e:
      ```
  private readonly ILogger<PollEmailProcessing> _logger;
        private readonly HttpClient _httpClient;

        public PollEmailProcessing(ILogger<PollEmailProcessing> logger, IHttpClientFactory httpClientFactory)
        {
            _logger = logger;
            _httpClient = httpClientFactory.CreateClient();
        }
```

Reply:
Done